src/dbnode/server/server.go

```diff
@@ -630,6 +633,21 @@ func interrupt() <-chan os.Signal {
 	return c
 }
 
+func bgValidateProcessLimits(logger xlog.Logger) {
```

Comment:
This is annoying, but should we add a signal to stop this goroutine at some point?  

Just because if we start using this more in integration tests having a dangling goroutine may become annoying to deal with.

Reply:
talked offline: added max monitor duration to control this